### PR TITLE
Fix typo in README

### PR DESCRIPTION
**Pull request by meme8383**

### CODE REVIEW

File: omega/analyzer/README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ docker run --rm --entrypoint /bin/bash --env-file .env openssf/omega-toolshed:la
 -->
 
 ### Local Usage
-There are currently 2 ways of getting the tar file to run the analyzer locally, through libraries.io or github. Here is a working example of one might have to wget to get it on the container. Then you can use the `?local=true` from [./worker/tools/runtools.sh](./worker/tools/runtools.sh) to check for packages. Naturally, as virtue of running it locally, you lose the ability to use `@latest` as the `<version_number>`in the package format (`pkg:npm/left-pad@<version_number>`) and need to explicitly write the verison number
+There are currently 2 ways of getting the tar file to run the analyzer locally, through libraries.io or github. Here is a working example of one might have to wget to get it on the container. Then you can use the `?local_source=true` from [./worker/tools/runtools.sh](./worker/tools/runtools.sh) to check for packages. Naturally, as virtue of running it locally, you lose the ability to use `@latest` as the `<version_number>`in the package format (`pkg:npm/left-pad@<version_number>`) and need to explicitly write the verison number
```
Comment (AOrps) [inline omega/analyzer/README.md:101]: Good catch!

Comment (AOrps) [reply omega/analyzer/README.md:101]: The only place, I see `local_source` being used as opposed to `local` is the directory filepath, but that only explains where it should be in the context of the a local download, not if it uses it.

Comment (AOrps) [reply omega/analyzer/README.md:101]: Interesting, you are clear. Good catch. Be sure to change the docs on runtools.sh as well.

I ran:
`$ ./runtools pkg:github.com/CycloneDX/cdxgen@9.7.4?local_source=true`
`$ ./runtools pkg:github.com/CycloneDX/cdxgen@9.7.4?local=true`

After doing:
`$ docker run --rm -it --entrypoint /bin/bash toolshed `

Comment (meme8383) [reply omega/analyzer/README.md:101]: Will do when I have a chance, this was causing problems for our team until I found this issue. We are also working on a fork to analyze GitHub repositories natively (https://github.com/purs3lab/alpha-omega).

Comment (AOrps) [reply omega/analyzer/README.md:101]: Ya ofc, It's a good find. So kudos!

Cool! That would be epic for github native repos, neat trick with using commit hash as version number. I just tested that and very clever! Thanks

**Pull request closed without merging**